source/vir/src/recursion.rs

```diff
@@ -466,13 +518,17 @@ pub(crate) fn expand_call_graph(
                 }
 
                 // f --> f2
-                call_graph.add_edge(f_node.clone(), Node::Fun(callee.clone()))
+                call_graph.add_edge(f_node.clone(), Node::Fun(callee.clone()));
+                // Ordering (+a): also add an edge f --> f2' where f2' is the method implemented by f2
+                if let CallTargetKind::Method(Some((_, _, _))) = kind {
+                    call_graph.add_edge(f_node.clone(), Node::Fun(x.clone()));
```

Comment:
With https://github.com/verus-lang/verus/pull/917, I think this should no longer be needed because we'd already have `x_resolved --> T --> x`.